targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F413xH/device/TOOLCHAIN_IAR/startup_stm32f413xx.S

```diff
@@ -1,8 +1,6 @@
 ;/******************** (C) COPYRIGHT 2017 STMicroelectronics ********************
 ;* File Name          : startup_stm32f413xx.s
 ;* Author             : MCD Application Team
-;* Version            : V2.6.1
```

Comment:
Curious, was there a particular reason why the version and date fields were removed?

Reply:
Hi
I aligned file with the ST HAL cube version.
Drivers team were incrementing this version number for each maintenance release, even if there was no change in files.
I understood they have changed their process.